### PR TITLE
[doc/formal] remove local run script from README

### PR DESCRIPTION
**Pull request by cindychip**
This PR removes the local script to run FPV and CONN. Right now we only
support the dvsim method.
This PR also cleans up the code that did not follow one sentence per line rule.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

File: hw/formal/README.md [hunk 1]
```diff
@@ -329,26 +333,7 @@ Connectivity tests use formal method to exhaustively verify system-level
...
-* `-p {core_path}`: specify which fusesoc core file to use. Default is
-  `lowrisc:systems:top_earlgrey`
-* `-gui`: run the connectivity script with GUI mode
-* `-cov`: collect connectivity test coverage. TODO: currently coverage takes a
-  long time
-* `-t {tool}`: specify which tool to run. Default is `JasperGold` and currently
-  only support JasperGold
-
-#### DVSim
+### Cadence JapserGold on dvsim
```
Comment (msfschaffner) [inline hw/formal/README.md:336]: Japser -> Jasper

File: hw/formal/README.md [hunk 2]
```diff
@@ -252,23 +252,27 @@ can be used to add coverpoints to your design, where the cover
...
 
 ### Cadence JapserGold
 If you have access to JasperGold from Cadence, you can formally verify your
-assertions. For example, to run formal property verification (FPV) using
+assertions.
+Before running FPV, please make sure the target has been added to the [batch script](
+https://github.com/lowRISC/opentitan/blob/master/hw/top_earlgrey/formal/top_earlgrey_fpv_cfgs.hjson)
+
+For example, to run formal property verification (FPV) using
 JasperGold on module `gpio`, type:
```
Comment (msfschaffner) [inline hw/formal/README.md:264]: can you format this as one sentence per line?

**Review by msfschaffner — APPROVED** [review]
LGTM with two nits.

File: hw/formal/README.md (hunk 2 above)
Comment (cindychip) [reply hw/formal/README.md:264]: Thanks for catching that!

File: hw/formal/README.md (hunk 1 above)
Comment (msfschaffner) [reply hw/formal/README.md:263]: looks like this is still here ;)

Comment (cindychip) [reply hw/formal/README.md]: oops my bad, did not push it correctly.

**Review by msfschaffner — APPROVED** [review]

**Comment by imphil** [comment]
Thanks Cindy!

**Review by weicaiyang — APPROVED** [review]
LGTM

**Pull request merged**